code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -561,7 +561,7 @@
 	projectile_type = /obj/item/projectile/bullet/rifle/a556mm
 
 /obj/item/projectile/bullet/rifle/a556mm
```

Comment:
Remove this. This alters damage values for all 5.56 rifles. The Pulse Rifle shouldn't use this bullet anyway.